js/intro.js

```diff
@@ -64,7 +64,7 @@ function generateNoDataFoundPage() {
   notFound.appendChild(notFoundDiv);
 }
 
-function generateSavedDetailsForm(users) {
+function generateSavedDetailsForm(users, dateDiff = NaN) {
```

Comment:
`generateSavedDetailsForm` => `generateSavedDetailsFrom`
As we are generating saved Details **FROM** the passed arguments.